css/css-sizing/aspect-ratio/META.yml

```diff
@@ -1,4 +1,159 @@
+  - test: block-aspect-ratio-007.html
+  - test: block-aspect-ratio-008.html
+  - test: block-aspect-ratio-009.html
+  - test: block-aspect-ratio-010.html
+  - test: block-aspect-ratio-011.html
+  - test: block-aspect-ratio-012.html
+  - test: block-aspect-ratio-013.html
+  - test: block-aspect-ratio-014.html
+  - test: block-aspect-ratio-015.html
+  - test: block-aspect-ratio-016.html
+  - test: block-aspect-ratio-017.html
+  - test: block-aspect-ratio-018.html
+  - test: block-aspect-ratio-019.html
+  - test: block-aspect-ratio-020.html
+  - test: block-aspect-ratio-021.html
+  - test: block-aspect-ratio-022.html
+  - test: block-aspect-ratio-023.html
+  - test: block-aspect-ratio-024.html
+  - test: block-aspect-ratio-025.html
+  - test: block-aspect-ratio-026.html
+  - test: block-aspect-ratio-027.html
+  - test: block-aspect-ratio-028.html
+  - test: block-aspect-ratio-029-crash.html
+  - test: block-aspect-ratio-030.html
+  - test: block-aspect-ratio-with-margin-collapsing-001.html
+  - test: block-aspect-ratio-with-margin-collapsing-002.html
+  - test: flex-aspect-ratio-001.html
+  - test: flex-aspect-ratio-002.html
+  - test: flex-aspect-ratio-003.html
+  - test: flex-aspect-ratio-004.html
+  - test: flex-aspect-ratio-005.html
+  - test: flex-aspect-ratio-006.html
+  - test: flex-aspect-ratio-007.html
+  - test: flex-aspect-ratio-008.html
+  - test: flex-aspect-ratio-009.html
+  - test: flex-aspect-ratio-010.html
+  - test: flex-aspect-ratio-011.html
+  - test: flex-aspect-ratio-012.html
+  - test: flex-aspect-ratio-013.html
+  - test: flex-aspect-ratio-014.html
+  - test: flex-aspect-ratio-015.html
+  - test: flex-aspect-ratio-016.html
+  - test: flex-aspect-ratio-017.html
+  - test: flex-aspect-ratio-018.html
+  - test: flex-aspect-ratio-019.html
+  - test: flex-aspect-ratio-020.html
+  - test: flex-aspect-ratio-021.html
+  - test: flex-aspect-ratio-022.html
+  - test: flex-aspect-ratio-023.html
+  - test: flex-aspect-ratio-024.html
+  - test: flex-aspect-ratio-025.html
+  - test: flex-aspect-ratio-026.html
+  - test: grid-aspect-ratio-001.html
+  - test: grid-aspect-ratio-002.html
+  - test: grid-aspect-ratio-003.html
+  - test: grid-aspect-ratio-004.html
+  - test: grid-aspect-ratio-005.html
+  - test: grid-aspect-ratio-006.html
+  - test: grid-aspect-ratio-007.html
+  - test: grid-aspect-ratio-008.html
+  - test: grid-aspect-ratio-009.html
+  - test: grid-aspect-ratio-010.html
+  - test: grid-aspect-ratio-011.html
+  - test: grid-aspect-ratio-012.html
+  - test: grid-aspect-ratio-014.html
+  - test: intrinsic-size-001.html
+  - test: intrinsic-size-002.html
+  - test: intrinsic-size-003.html
+  - test: intrinsic-size-004.html
+  - test: intrinsic-size-005.html
+  - test: intrinsic-size-006.html
+  - test: intrinsic-size-007.html
+  - test: intrinsic-size-008.html
+  - test: intrinsic-size-009.html
+  - test: intrinsic-size-010.html
+  - test: intrinsic-size-011.html
+  - test: parsing/aspect-ratio-computed.html
```

Comment:
Good, this hit https://github.com/web-platform-tests/wpt-metadata/blob/0064a416666d3b0e1194f9ba3bbd5f9d7efbb2b2/go_test/parse_test.go#L57. Fixing...